src/app/board/components/tasks/tasks-list/tasks-list.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, Input } from '@angular/core';
+import { ITask } from '../../../models/task.model';
+
+@Component({
+  selector: 'app-tasks-list',
+  templateUrl: './tasks-list.component.html',
+  styleUrls: ['./tasks-list.component.scss'],
+})
+export class TasksListComponent {
+  @Input() tasks: ITask[] | undefined;
```

Comment:
is it not ok to set initial value [] for this Input?

Reply:
> is it not ok to set initial value [] for this Input?

Done